src/ScottPlot5/ScottPlot5 Tests/Unit Tests/UnitTests/PrimitiveTests/CoordinateRectTests.cs

```diff
@@ -38,4 +38,20 @@ public void Test_CoordinateRect_Expanded()
         CoordinateRect cRect2 = cRect.Expanded(coord);
         cRect2.Contains(coord).Should().BeTrue();
     }
+
+    [TestCase(0, 0, 1, 1, 0.5, 0.5, true, TestName = "Test Contains with no inverted")]
+    [TestCase(0, 0, 1, -1, 0.5, -0.5, true, TestName = "Test Contains with inverted X")]
+    [TestCase(0, 0, -1, 1, -0.5, 0.5, true, TestName = "Test Contains with inverted Y")]
+    [TestCase(0, 0, -1, -1, -0.5, -0.5, true, TestName = "Test Contains with inverted X and Y")]
```

Comment:
These test cases are a little hard to parse, just because there's 6 numbers in there, I'd prefer to pass them in as a struct or something, just so that it's more clear to read.

I might suggest something like this:
```cs
private static readonly (CoordinateRect rect, Coordinates pt)[] ContainsTestCases = {
    (new(0, 1, 0, 1), new(0.5, 0.5)), // Uninverted Rect
    (new(0, 1, 0, -1), new(0.5, -0.5)), // inverted X
    (new(0, -1, 0, 1), new(-0.5, 0.5)), // inverted Y
    (new(0, -1, 0, -1), new(-0.5, -0.5)) // inverted X and Y
};

[Test, TestCaseSource(nameof(ContainsTestCases))]
public void Test_CoordinateRect_Contains((CoordinateRect rect, Coordinates pt) testCase)
{
    Assert.That(testCase.rect.Contains(testCase.pt), Is.True);
}
```

Note that it's equivalent, the order of the numbers is just different because the `CordinateRect` constructor takes `left, right, bot, top` and your code lists them in the order `left, bot, right, top`